_docs/2.0.0/usage/schema.md

```diff
@@ -375,6 +384,21 @@ For hexadecimal value entries in the form *"0xFF"* are accepted as well.
 If `radix` is specified as anything other than the default 10, [pattern](#pattern)
 value will be ignored.
 
+### encoding
+
+<sup>**Supported by types:** [Binary](#Binary)</sup>
+
+**When a string value is being converted to binary**, the supplied `encoding` indicates how the values are going to be 
+treated. This applies for the default value, too:
+ - `none` - the input will get cast as-is to binary. E.g. "abc" -> [97, 98, 99]
+ - `base64` - the input is considered as Base64-encoded and will get unencoded. Contrary to the basic Spark behavior of
+ `unbase64` (which skips characters invalid for Base64), this will result in an error.
+
+If encoding is missing altogether would it be needed (e.g. default value is given), `ValidationWarning` is issued and
```

Comment:
I am not sure what this sentence is saying

Reply:
True, rephrased.